### PR TITLE
Jenkins scripting

### PR DESCRIPTION
**Pull request by coryan**
Creates a script to refactor all the Jenkins configs.  Also the script
can be kept in sync with the code, should we move or rename the
benchmarks again.

### CODE REVIEW

File: ci/build-jenkins.sh [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+ctest --output-on-failure
+
+case ${BENCHMARK} in
+    endurance)
+        bigtable/benchmarks/endurance_benchmark ${PROJECT_ID} ${INSTANCE_ID} 4 82800;
+        ;;
+    latency)
+        bigtable/benchmarks/apply_read_latency_benchmark ${PROJECT_ID} ${INSTANCE_ID};
+        ;;
+    throughput)
```
Comment (sduskis) [inline ci/build-jenkins.sh]: Can you please add a comment here about why 60 is used here?  Specifically that our tests run on a 4 CPU machine and that 15 threads per CPU seems to saturate 1 CPU.

Comment (coryan) [reply ci/build-jenkins.sh]: Will do.  We should talk about the actual number of threads though, my experiments required more like 128 threads to get 80% CPU utilization on a n1-standard-4 instance.

Comment (coryan) [reply ci/build-jenkins.sh]: And let's discuss this and fine tune the number before I merge the code.

Comment (coryan) [reply ci/build-jenkins.sh]: Just to keep you informed: after growing the cluster to 6 nodes, and if we include the upgrade to gRPC-v1.10.x (#283, it is pending review) I managed to get the CPU to 100% (or close, 99.x%) with 90 threads, with 60 I am getting ~89%, trying 75 threads right now to see how that does.

**Pull request merged**